packer/grapl-service/files/consul.conf.json

```diff
@@ -0,0 +1,10 @@
+{
+	 "server": false
+	,"datacenter": "my-little-datacenter"
```

Comment:
We'll probably want to create templates for these files, rather than baking in values like this.